readme.md

```diff
@@ -5,7 +5,7 @@
 **Tags:**              [admin](https://wordpress.org/plugins/tags/admin/), [posts](https://wordpress.org/plugins/tags/posts/), [users](https://wordpress.org/plugins/tags/users/)  
 **Requires at least:** 4.6  
 **Tested up to:**      4.8  
-**Stable tag:**        1.0.0  
+**Stable tag:**        0.2.0  
```

Comment:
Should this be 1.2.0?

Reply:
see @EvanHerman commit https://github.com/godaddy/wp-reseller-store/pull/42/commits/47242ec0bfa12d3aabe9ebdd976c33846dd82655